parser-typechecker/src/Unison/Runtime/IOSource.hs

```diff
@@ -150,8 +150,8 @@ type Either a b = Left a | Right b
 
 type Optional a = None | Some a
 
-d1 Doc.++ d2 = 
-  use Doc 
+d1 Doc.++ d2 =
+  use Doc
```

Comment:
I could have sworn I saw something on slack about trailing whitespace, my vim config does this, sorry.